plugins/cilium-docker/driver/driver.go

```diff
@@ -414,7 +414,7 @@ func (driver *driver) createEndpoint(w http.ResponseWriter, r *http.Request) {
 	}
 	if err != nil {
 		sendError(w,
-			fmt.Sprintf("Error while setting up %s mode: %s", driver.conf.DatapathMode, err),
```

Comment:
I would suggest to change the `drive.conf.DatapathMode` to a variable that is assigned for each case in the switch-case statement above.

Reply:
Thanks, done.